packages/turf-helpers/index.ts

```diff
@@ -66,8 +66,8 @@ export type Corners = "sw" | "se" | "nw" | "ne" | "center" | "centroid";
 
 export type Lines = LineString | MultiLineString | Polygon | MultiPolygon;
 export type AllGeoJSON =
-  | Feature
-  | FeatureCollection
+  | Feature<Geometries | GeometryCollection>
+  | FeatureCollection<Geometries | GeometryCollection>
```

Comment:
This addition is to overcome the `geojson` package not providing a default interface for the generic, the user is left to specify it.  Turf had previously specified one.

geojsonDT:
```
export interface Feature<G extends GeometryObject, P = GeoJsonProperties> 
```

turf (previously):
```
export interface Feature<G = Geometry | GeometryCollection, P = Properties>
```

Reply:
I think this default interface setting is a candidate to contribute upstream to `@types/geojson`

Reply:
This is out of date, reworking

Reply:
The latest `geojson` types already provide a default, turf was somehow using an older version, resolving.